options.go

```diff
@@ -120,8 +121,29 @@ func HashRingConfig(c *hashring.Configuration) Option {
 // automatically.
 func Logger(l log.Logger) Option {
 	return func(r *Ringpop) error {
-		r.logger = l
-		r.log = l
+		if r.moduleLogger == nil {
+			r.moduleLogger = rutil.NewModuleLogger(l)
+		} else {
+			r.moduleLogger.SetLogger(rutil.RootModule, l)
+		}
+		return nil
+	}
+}
+
+// ModuleLevel is used to configure different log levels per module. Messages
+// with a level less than minLevel are silenced. By default, an unconfigured
+// module outputs all messages.
+func ModuleLevel(name string, minLevel rutil.Level) Option {
+	return func(rp *Ringpop) error {
+		rp.moduleLogger.SetLevel(name, minLevel)
+		return nil
+	}
+}
+
+// ModuleLogger is used to configure different loggers per module.
```

Comment:
Can we include examples here? Note that this godoc is important, since it will be displayed in the ringpop-go documentation and seen by our customers.